### PR TITLE
config number of rows to add when we click on add Button

### PR DESCRIPTION
**Pull request by gudipudipradeep**
#134 Add default rows to add from configuration of sheet

### CODE REVIEW

File: packages/core/src/context.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ export type Context = {
   luckysheetfile: Sheet[];
   defaultcolumnNum: number;
   defaultrowNum: number;
+  adddefaultrows: number;
```
Comment (shuanhukexuejia) [inline packages/core/src/context.ts:23]: It's better to use lowerCamelCase when adding new arguments

File: packages/core/src/context.ts [hunk 2]
```diff
@@ -176,6 +177,7 @@ export function defaultContext(): Context {
     luckysheetfile: [],
     defaultcolumnNum: 60,
     defaultrowNum: 84,
+	adddefaultrows: 50,
```
Comment (shuanhukexuejia) [inline packages/core/src/context.ts:180]: Lint errors, try running ```yarn lint --fix``` to see if it works.

File: packages/core/src/context.ts (hunk 1 above)
Comment (gudipudipradeep) [reply packages/core/src/context.ts:23]: Am planning to rename the variable to addDefaultRows. Please suggest me

Comment (shuanhukexuejia) [reply packages/core/src/context.ts:23]: That's right

File: packages/core/src/context.ts (hunk 2 above)
Comment (gudipudipradeep) [reply packages/core/src/context.ts]: Fixed Lint issue and Change the variable as lowerCamelCase. @shuanhukexuejia. Please review.

**Comment by shuanhukexuejia** [comment]
The screenshot is referenced in README
image

**Review by shuanhukexuejia — APPROVED** [review]

**Comment by jikoczl** [comment]
There are two commits in this PR. You can fixup them into one commit with

**Comment by jikoczl** [comment]
@gudipudipradeep hi, It seems like you are not familiar with git rebase. Do you need some help?

**Comment by gudipudipradeep** [comment]
@jikoczl  yes, Am facing some difficulty to do that. Please help me

**Comment by jikoczl** [comment]
@gudipudipradeep 
give a screen shot of "git log", I want to see the commit history.

**Comment by gudipudipradeep** [comment]
image
@jikoczl this git log

**Comment by jikoczl** [comment]
firstly, $ git rebase -i e7986ee.
Then edit in vim, don't change the first line.Change the 2nd-5th line, change pick to f.
Esc, :wq!, Enter

**Comment by gudipudipradeep** [comment]
@jikoczl this is the screen shot after above instructions.
image

**Comment by jikoczl** [comment]
@gudipudipradeep, It seems to be right.
Then, git commit --amend. Edit the commit message in vim

**Comment by jikoczl** [comment]
@gudipudipradeep, Can you give me the write access to your fork. In case some thing wrong happen,then I can directly do something in your branch.

**Comment by jikoczl** [comment]
@gudipudipradeep , Why every time after you push your change, there will be two new commit? What command do you use to push changes?

**Comment by gudipudipradeep** [comment]
@jikoczl  Am using GitHub Desktop while pushing changes

**Comment by jikoczl** [comment]
@gudipudipradeep , Don't use that, just use 'git push --force' .
I will give you a new solution.

$ git reset e7986ee --soft
$ git commit -m 'feat: xxxxx'
$ git push --force

**Comment by jikoczl** [comment]
@gudipudipradeep,hey bro, It seems like you have a huge problem now. I give a PR of your changes, check this whehter it is right. #148

**Pull request closed without merging**